tests/testthat/test-col_identify.R

```diff
@@ -0,0 +1,46 @@
+test_that("col_identify should return a datafame", {
```

Comment:
Right now, test function of col_identify is a separate function. You need to move it into test-collinearityR script.